includes/class-sensei-course.php

```diff
@@ -2810,6 +2810,19 @@ public static function flush_rewrite_rules( $post_id ){
 
 	}
 
+	/**
+	 * Optionally return the full content on the single course pages
+	 * depending on the users course_single_content_display setting
+	 *
+	 * @deprecated since 1.11.0
+	 * @param $excerpt
+	 * @return string
+	 */
+	public static function full_content_excerpt_override( $excerpt ) {
+		_deprecated_function( __METHOD__, '1.11.0', 'The single course page display setting is no longer used.' );
```

Comment:
1.11.0 -> 1.12.0

This will generate the following PHP notice:
```
PHP Notice:  Sensei_Course::full_content_excerpt_override is <strong>deprecated</strong> since version 1.11.0! Use The single course page display setting is no longer used. instead
```
The last parameter is meant to provide the name of the new function to call instead. Since there isn't one in this case, we can leave the last parameter off.